### DISCUSSION TITLE
Loading and Editing a model

### DISCUSSION THREAD

**Discussion by Rahmaabubakr** [opencobra/cobrapy#1360] [Q&A]
Hello Cobrapy Community,

Hope everyone is doing well. I would really appreciate your capacity to answer my basic questions since I am new to Cobrapy. First, how can I load a model from the computer? how can i edit the id names of some metabolites since they contain numbers, should i open the file and just change it?

Thank you very much,
Rahma A. Abubakr

**Comment by Rahmaabubakr** [comment]
I would really appreciate getting answers here, please :)

**Comment by cdiener** [comment]
Hi,

It's a pretty general question. There are many ways to load models with COBRAPY. They are documented at https://cobrapy.readthedocs.io/en/latest/io.html . You can change metabolite names by setting their id (like `model.metabolites[0].id = ...`). But it's not necessarily required, you can access metabolites whose IDs start with numbers perfectly fine as well. For instance, if your metabolite is named "1,3-abc" `model.metabolites.1,3-abc` will not work but `model.metabolites.get_by_id("1,3-abc")` will work just fine.

**Reply by Rahmaabubakr** [reply]
Hello,

Thank you very much for your answer.

It is working fine with model.metabolites.get_by_id ("...")

but with this  model.metabolites.("2ahhmp_LSQBKTc_RSQBKT").summary()  command it is not working

it gives this error

 Cell In[30], line 1
    model.metabolites.("2ahhmp_LSQBKTc_RSQBKT").summary()
                     ^
SyntaxError: invalid syntax

**Reply by matthiaskoenig** [reply]
you forgot the `get_by_id`
